spec/lib/dor/was_seed/thumbnail_generator_service_spec.rb

```diff
@@ -3,60 +3,64 @@
+      described_class.capture_thumbnail(druid, workspace, uri)
+      expect(File.exist?(thumbnail_jp2_file)).to be true
+      thumbnail_image = Vips::Image.new_from_file(thumbnail_jp2_file)
+      expect(thumbnail_image.width).to eq 400
+      expect(thumbnail_image.height).to eq 400
+      expect(File.exist?(screenshot_jpeg_file)).to be false # screenshot jpeg is removed - it's produced on the way to the jp2
     end
 
     it 'raises an error if the screenshot method raises an exception' do
       allow(described_class).to receive(:screenshot).and_raise('Foo')
       exp_msg = "Thumbnail for druid druid:ab123cd4567 and http://www.slac.stanford.edu can't be generated.\n Foo"
-      expect { described_class.capture_thumbnail(@druid_id, @workspace, @uri) }.to raise_error.with_message(exp_msg)
-      expect(File.exist?('spec/was_seed_preassembly/fixtures/workspace/ab/123/cd/4567/ab123cd4567/content/thumbnail.jp2')).to be false
-      expect(File.exist?('tmp/ab123cd4567.jpeg')).to be false
+      expect { described_class.capture_thumbnail(druid, workspace, uri) }.to raise_error.with_message(exp_msg)
+      expect(File.exist?(thumbnail_jp2_file)).to be false
+      expect(File.exist?(screenshot_jpeg_file)).to be false
     end
   end
 
   describe '.screenshot' do
-    before do
-      FileUtils.rm 'tmp/test_capture.jpeg', force: true
-    end
-
-    after do
-      FileUtils.rm 'tmp/test_capture.jpeg', force: true
-    end
-
-    it 'captures jpeg image for the first capture of url', :image_prerequisite do
-      # FIXME: We have NO TEST EXECUTING FOR THIS METHOD!
-      pending 'fails from lack of VCR cassette and maybe also jquery error'
-      VCR.use_cassette('slac_capture') do
-        wayback_uri = 'https://swap.stanford.edu/20110202032021/http://www.slac.stanford.edu'
-        temporary_file = 'tmp/test_capture'
-        result = described_class.screenshot(wayback_uri, temporary_file)
-        expect(result).to eq('')
-      end
-    end
+    skip('to test .screenshot method we need to execute puppeteer in a headless browser')
+    # let(:wayback_uri) { "https://swap.stanford.edu/#{Dor::WasSeed::ThumbnailGeneratorService::DATE_TO_TRIGGER_EARLIEST_CAPTURE}/http://www.slac.stanford.edu" }
+    # let(:screenshot_jpeg_file) { 'tmp/test_capture.jpeg' }
+    #
+    # before do
+    #   FileUtils.rm screenshot_jpeg_file, force: true
+    #   FileUtils.cp 'spec/was_seed_preassembly/fixtures/thumbnail_files/ab123cd4567.jpeg', screenshot_jpeg_file
+    # end
+    #
+    # after do
+    #   FileUtils.rm screenshot_jpeg_file, force: true
+    # end
+    #
+    # it 'captures jpeg image for the first capture of url' do
+    #   allow(described_class).to receive(:screenshot)
+    #   described_class.screenshot(wayback_uri, screenshot_jpeg_file)
+    #   screenshot_image = Vips::Image.new_from_file(screenshot_jpeg_file)
+    #   expect(screenshot_image.width).to eq 1000
+    #   expect(screenshot_image.height).to eq 1000
+    # end
```

Comment:
let's state exactly why this test is skipped (instead of "image_prerequisite") and comment out the test code to double up on clarity of "NOT EXECUTED"